tests/unit_tests/simtel/test_simtel_config_writer.py

```diff
@@ -171,3 +187,26 @@ def test_get_sim_telarray_metadata_without_model_parameters(simtel_config_writer
 
     with pytest.raises(ValueError, match=r"^Unknown metadata type"):
         simtel_config_writer._get_sim_telarray_metadata("unknown", None)
+
+
+def test_write_random_seeds_file(simtel_config_writer, tmp_test_directory):
+    config_file_directory = Path(tmp_test_directory) / "config"
```

Comment:
This would be a non-standard structure of the testing directory, no? I would expect this file to be written to the model directory and that should already exist.

Reply:
Changed it to `model`. You are correct that the directory exists, but in a unit tests it might be better to not depend on directories made in code outside of the tested one. Anyway - for the test not so important.

Reply:
It's created as part of the `tmp_test_directory` fixture, so it's probably safe (or changes would be required in many places anyway). Either way, thanks!